mock/src/mock/account.rs

```diff
@@ -42,10 +42,24 @@ pub fn mock_account_storage() -> AccountStorage {
     AccountStorage::new(vec![storage_item_0(), storage_item_1()]).unwrap()
 }
 
-// Constants that define the indexes of the account procedures of interest
-pub const ACCOUNT_PROCEDURE_INCR_NONCE_PROC_IDX: usize = 2;
-pub const ACCOUNT_PROCEDURE_SET_ITEM_PROC_IDX: usize = 3;
-pub const ACCOUNT_PROCEDURE_SET_CODE_PROC_IDX: usize = 4;
+// The MAST root of the default account's interface. Use these constants to interact with the
+// account's procedures.
+pub const ACCOUNT_RECEIVE_ASSET_MAST_ROOT: &str =
+    "0x5e2981e93f961ddc63f5f65332554cafabe642d1eccff58a9279be4a13075f15";
+pub const ACCOUNT_SEND_ASSET_MAST_ROOT: &str =
+    "0xc5740bad4e353fda39044d29436beb899629667fc57381aaf7971d664712ef5c";
+pub const ACCOUNT_INCR_NONCE_MAST_ROOT: &str =
+    "0xd765111e22479256e87a57eaf3a27479d19cc876c9a715ee6c262e0a0d47a2ac";
+pub const ACCOUNT_SET_ITEM_MAST_ROOT: &str =
+    "0xf7f1a1facd65a56bda0471acaf12c62b99d9f9c8e33c9fa02a64b167fb7669db";
+pub const ACCOUNT_SET_CODE_MAST_ROOT: &str =
+    "0x9d221abcc386973775499406d126764cdf4530ccf8084e27091f7e9f28177bbe";
+pub const ACCOUNT_CREATE_NETWORK_NOTE_MAST_ROOT: &str =
+    "0x9b90ba0e1af786b3a2c618ec9fb53c5441610cd7d7aec26160300cf9bd3762eb";
```

Comment:
Finding this MAST took me a surprisingly long time because we have so many procedures with the same name:

- The account has a `create_network_note`
- The tx lib has a `create_network_note`
- The kernel has a `create_network_note`

All of the above MASTs are different. On top of that, the proc_id is different than the MAST, and the assembler doesn't seem to keep track of these elements together, so it is a bit of work to extract these details from there. I hope with the latest changes of the assembler this will be a bit easier, but I think we should reevalute how we are tracking methods and streamline this.

For now, I think this hack should suffice. I hope the comments will guide people that are unfamiliar with this complexity when these values need to be updated.